modules/fargate/README.md

```diff
@@ -15,6 +15,7 @@ See example code in `examples/fargate`.
 | name | Fargate profile name | `string` | Auto generated in the following format `[cluster_name]-fargate-[fargate_profile_map_key]`| no |
 | selectors | A list of Kubernetes selectors. See examples/fargate/main.tf for example format. | <pre>list(map({<br>namespace = string<br>labels = map(string)<br>}))</pre>| `[]` | no |
 | subnets | List of subnet IDs. Will replace the root module subnets. | `list(string)` | `var.subnets` | no |
+| timeouts | A map of timeouts for create/delete operations. | `map(string)` | Provider default behavior | no |
```

Comment:
```suggestion
| timeouts | A map of timeouts for create/delete operations. | `map(string)` | Provider default behaviour | no |
```

Reply:
US vs UK :) Let's leave the English language to native-language speakers (definitely, not me and you)